lib/connections/connections.go

```diff
@@ -295,7 +325,7 @@ func (s *connectionSvc) connect() {
 					s.model.Close(deviceID, fmt.Errorf("switching connections"))
 				}
 
-				s.conns <- model.IntermediateConnection{
+				s.Conns <- model.IntermediateConnection{
```

Comment:
Is this the channel you're concerned about crossing packages? From connections to discover?

Assuming so, I'll work on changing it.